### PR TITLE
#111 - fix login fail error message disappearence - Aisha Athman Lali

### PR DESCRIPTION
**Pull request by aishaathmanlali**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/TenderClient.js [hunk 1]
```diff
@@ -21,7 +21,9 @@ const handleAuthError = (error) => {
 	if (error.response && error.response.status === 401) {
 		localStorage.removeItem("authToken");
 		localStorage.removeItem("userType");
-		window.location.href = "/";
+		if (window.location.href !== "/") {
+			window.location.href !== "/";
+		}
```
Comment (soonick) [inline client/src/TenderClient.js]: This is incorrect. Please test your code before sending it for review

**Review by soonick — CHANGES_REQUESTED** [review]

File: client/src/TenderClient.js (hunk 1 above)
Comment (aishaathmanlali) [reply client/src/TenderClient.js:26]: i did test it and it was working

File: client/src/TenderClient.js [hunk 2]
```diff
@@ -21,7 +21,9 @@ const handleAuthError = (error) => {
 	if (error.response && error.response.status === 401) {
 		localStorage.removeItem("authToken");
 		localStorage.removeItem("userType");
-		window.location.href = "/";
+		if (window.location.href !== "/") {
+			window.location.href === "/";
```
Comment (soonick) [inline client/src/TenderClient.js]: Again. Test your code before sending it for review. This is still incorrect

**Review by soonick — CHANGES_REQUESTED** [review]

File: client/src/TenderClient.js (hunk 2 above)
Comment (aishaathmanlali) [reply client/src/TenderClient.js]: I have tested it

Comment (aishaathmanlali) [reply client/src/TenderClient.js]: its working

Comment (soonick) [reply client/src/TenderClient.js]: You are testing only the scenario when the if is false. You are not testing the scenario when the if is true

Comment (soonick) [reply client/src/TenderClient.js]: You still haven't tested the scenario where the if is true. Please test it

**Review by soonick — CHANGES_REQUESTED** [review]

File: client/src/TenderClient.js (hunk 2 above)
Comment (soonick) [reply client/src/TenderClient.js]: @aishaathmanlali , it's still incorrect

**Review by soonick — CHANGES_REQUESTED** [review]

**Review by soonick — APPROVED** [review]

**Pull request merged**